static/app/views/alerts/create.spec.jsx

```diff
@@ -93,28 +97,34 @@ describe('ProjectAlertsCreate', function () {
     };
   };
 
-  it('adds default parameters if wizard was skipped', function () {
+  it('adds default parameters if wizard was skipped', async function () {
     const location = {query: {}};
     const wrapper = createWrapper(undefined, location);
-
-    expect(wrapper.router.replace).toHaveBeenCalledWith({
-      pathname: '/organizations/org-slug/alerts/new/metric',
-      query: {
-        aggregate: 'count()',
-        dataset: 'events',
-        eventTypes: 'error',
-        project: 'project-slug',
-      },
+    await waitFor(() => {
+      expect(wrapper.router.replace).toHaveBeenCalledWith({
+        pathname: '/organizations/org-slug/alerts/new/metric',
+        query: {
+          aggregate: 'count()',
+          dataset: 'events',
+          eventTypes: 'error',
+          project: 'project-slug',
+        },
+      });
     });
   });
 
   describe('Issue Alert', function () {
-    it('loads default values', function () {
+    it('loads default values', async function () {
       createWrapper();
-
-      expect(screen.getByText('All Environments')).toBeInTheDocument();
-      expect(screen.getAllByDisplayValue('all')).toHaveLength(2);
-      expect(screen.getByText('30 minutes')).toBeInTheDocument();
+      await waitFor(() => {
+        expect(screen.getByText('All Environments')).toBeInTheDocument();
+      });
```

Comment:
should be able to use findBy on the first one and leave the rest as getBy
```suggestion
        expect(await screen.findByText('All Environments')).toBeInTheDocument();
```